featurebyte/models/base.py

```diff
@@ -116,7 +115,7 @@ def json_dict(self, **kwargs: Any) -> dict[str, Any]:
         -------
         dict[str, Any]
         """
-        output: dict[str, Any] = json.loads(self.model_dump_json(by_alias=True, **kwargs))
+        output: dict[str, Any] = self.model_dump(mode="json", by_alias=True, **kwargs)
```

Comment:
@gee-senbong Could you confirm this change is fine?

Reply:
Testing it locally, seems like it should be fine. Good to know such option exists.